src/bun.js/api/bun/subprocess.zig

```diff
@@ -2042,11 +2042,6 @@ pub const Subprocess = struct {
                 while (cmds_array.next()) |value| {
                     const arg = value.getZigString(globalThis);
 
-                    // if the string is empty, ignore it, don't add it to the argv
```

Comment:
Nice find, I also found this too but it got lost in some other commit. Mind extracting this out into another PR so it doesn't get lost again?

Reply:
Sure, though ideally #8981 would get merged first :-)